internal/dynamicanalysis/analysis.go

```diff
@@ -28,6 +32,24 @@ var resultError = &Result{
 	},
 }
 
+func ParseURLsFromSSlStripOutput(content []byte) []string {
+	var result []string
+	ret := strings.Split(string(content), "\n")
+
+	for _, value := range ret {
+		lineParts := strings.Split(value, " ")
+		if len(lineParts) < 11 {
+			continue
+		}
+
+		schema := lineParts[3]
+		host := lineParts[8]
+		path := lineParts[10]
+		result = append(result, schema+"://"+host+path)
```

Comment:
It could be useful having an example line from the log here so readers can understand what is being parsed.